test/unit-test/cmock/create_test.cmake

```diff
@@ -151,9 +152,9 @@ function(create_real_library target
             ${real_include_list}
         )
     set_target_properties(${target} PROPERTIES
-                COMPILE_FLAGS "-Wextra -Wpedantic \
+                COMPILE_FLAGS "-O0 -Wextra -Wpedantic \
                     -fprofile-arcs -ftest-coverage -fprofile-generate \
-                    -Wno-unused-but-set-variable"
+                    -Wno-unused-but-set-variable -ggdb"
```

Comment:
these you can add them only with debug builds
in case we want to add some speed tests you want to be with O3
cmake -DCMAKE_BUILD_TYPE=Release/Debug
check: CMAKE_C_FLAGS_[DEBUG|RELEASE|RELWITHDEBINFO|MINSIZEREL|...].

Reply:
Removed